packages/ComponentLibrary/src/components/DragModal/DragModalContent.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useRef } from 'react';
+import List from '@mui/material/List';
+import {
+  DndContext,
+  MouseSensor,
+  TouchSensor,
+  useSensor,
+  useSensors,
+  DragEndEvent,
+  UniqueIdentifier,
+  closestCenter,
+} from '@dnd-kit/core';
+import {
+  SortableContext,
+  arrayMove,
+  verticalListSortingStrategy,
+} from '@dnd-kit/sortable';
+import SortableItem from './SortableItem';
+import { DragModalContentProps } from './DragModal.types';
+import ModalDivider from '../ModalDivider';
+import DragIndicator from '../../assets/icons/drag.svg';
+import NavigateBefore from '../../assets/icons/chevron-left.svg';
+import { styles, sx } from './DragModal.styles';
+import { Box, Button, Link } from '@mui/material';
+import { theme } from '../../theme';
+import {
+  restrictToParentElement,
+  restrictToVerticalAxis,
+} from '@dnd-kit/modifiers';
+
+const DragModalContent: React.FC<DragModalContentProps> = ({
+  people,
+  setPeople,
+  onBack,
+  backButtonText,
+  activateAllText,
+  deactivateAllText,
+  buttonText,
+}) => {
+  const sensors = useSensors(
+    useSensor(MouseSensor, { activationConstraint: { distance: 5 } }),
+    useSensor(TouchSensor, { activationConstraint: { distance: 5 } }),
+  );
+
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  const handleDragEnd = (event: DragEndEvent) => {
+    const { active, over } = event;
+    if (over && active.id !== over.id) {
+      setPeople(items => {
+        const oldIndex = items.findIndex(item => item.id === active.id);
+        const newIndex = items.findIndex(item => item.id === over.id);
+        return arrayMove(items, oldIndex, newIndex);
+      });
+    }
+  };
+
+  const handleToggleAll = () => {
+    const allActivated = people.every(person => person.isActive);
+    setPeople(prev =>
+      prev.map(person => ({ ...person, isActive: !allActivated })),
+    );
+  };
+
+  const handleToggle = (id: UniqueIdentifier) => {
+    setPeople(prev =>
+      prev.map(person =>
+        person.id === id ? { ...person, isActive: !person.isActive } : person,
+      ),
+    );
+  };
+
+  return (
+    <>
```

Comment:
Just curious, what is the reason for the <></> wrapping the whole JSX?

Reply:
JSX expression must have a parent, and this is just a Fragment to wrap it instead of use <React.Fragment></React.Fragment>